README.md

```diff
@@ -26,6 +26,9 @@ let props = {
   optionalNumber: ElementPropTypes.number,
   optionalString: ElementPropTypes.string,
   optionalColor: ElementPropTypes.color,
+  optionalProduct: ElementPropTypes.product,
+  optionalCategory: ElementPropTypes.category,
+  optionalImageAttributes: ElementPropTypes.imageAttributes,
```

Comment:
I think Chris want this prop to be called image, according to your Slack chat from yesterday.

Reply:
👍